integrations/nuance_pin/app/inference.py

```diff
@@ -181,7 +181,7 @@ def pre_process(self, img_reader) -> Compose:
                 ToTensord(
                     keys=image_key,
                 ),
-                ToDeviced(keys=image_key, device="cuda"),
+                ToDeviced(keys=image_key, device=self.device),
```

Comment:
This comment is specific about this line, but Line 181.
Please rebase, as the inference.py has been updated in a previous PR by me to make the App SDK compositable with MONAI Core 0.9.1 and newer; for this one there is a small change in the pre-processing to remove the use of ToTensord